src/search_algo.hpp

```diff
@@ -284,13 +290,20 @@ loadDbIndexFromDisk(TGlobalHolder       & globalHolder,
     if constexpr (seqan::sIsTranslated(TGlobalHolder::blastProgram))
     {
         // last value has sum of lengths
-        seqan::context(globalHolder.outfileBlastTab).dbTotalLength  = globalHolder.indexFile.origSeqLengths.back();
-        seqan::context(globalHolder.outfileBlastTab).dbNumberOfSeqs = globalHolder.indexFile.origSeqLengths.size() - 1;
+        // seqan::context(globalHolder.outfileBlastTab).dbTotalLength  = globalHolder.indexFile.origSeqLengths.back();
+        // seqan::context(globalHolder.outfileBlastTab).dbNumberOfSeqs = globalHolder.indexFile.origSeqLengths.size() - 1;
+        seqan::context(globalHolder.outfileBlastTab).dbTotalLength  =
+            std::accumulate(globalHolder.indexFile.seqs.begin(), globalHolder.indexFile.seqs.end(), 0,
+                [](int sum, std::vector<typename TGlobalHolder::TOrigSbjAlph> a)
```

Comment:
```suggestion
                [](size_t sum, std::vector<typename TGlobalHolder::TOrigSbjAlph> const & a)
```

Please do not copy every sequence when adding their sizes :scream: